introduction.Rmd

```diff
@@ -184,6 +184,309 @@ We call the combination of these two steps the __quote and unquote__ pattern. Th
 Most of the time though, we don't need to create blueprints manually. We'll get them by quoting the arguments supplied by users. This gives your functions the same usage and feel as tidyverse verbs.
 
 
+# Do you need tidy eval? {#sec:do-you-need}
+
+In the field of computer science, frameworks like tidy evaluation are known as metaprogramming. Modifying the blueprints of computations amounts to programming the program, i.e. metaprogramming. In other languages, this type of approach is often seen as a last resort because it requires new skills and might make your code harder to read. Things are different in R because of the importance of data masking functions, but it is still good advice to consider other options before turning to tidy evaluation. In this section, we review several strategies for solving programming problems with tidyverse packages.
+
+Before diving into tidy eval, make sure to know about the fundamentals of programming with the tidyverse:
```

Comment:
Are you saying these are prerequisites to understanding tidy eval, or that they likely have a better return on investment of time? I _think_ it's the latter (basing this on @jennybc's talk). If so, perhaps reword (I can't think of anything good right now, but I'll work on it).

Reply:
Perhaps:

These are likely to have a better return on investment of time and will also be useful to your programming tasks outside the tidyverse.

Reply:
I like it!